packages/sync-actions/src/utils/copy-empty-array-props.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @function copyEmptyArrayProps
+ * @deprecated Takes two objects and if there is Array props in oldObj which doesnt exist in newObj then it Copy name with empty value
```

Comment:
Why do you use `@deprecated` here?

Reply:
Sorry my bad, it should be description, fix in min 👍